lib/main.dart

```diff
@@ -1,17 +1,35 @@
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:google_fonts/google_fonts.dart';
 
-import './features/login/presentation/pages/home.dart';
+import './features/login/presentation/pages/login_options.dart';
 
 void main() {
-  runApp(
-    MaterialApp(
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    // Prevents the app from changing orientation
+    SystemChrome.setPreferredOrientations([
+      DeviceOrientation.portraitUp,
+      DeviceOrientation.portraitDown,
+    ]);
+
+    return MaterialApp(
+      title: 'Swipe n Swap',
       debugShowCheckedModeBanner: false,
-      title: 'SwipeNSwap',
-      home: Scaffold(
-        body: SafeArea(
```

Comment:
I am guessing this safe area is transferred to the childredn. i.e. the hoime page? Right not the login does not go to home page correct but when it does it will inheirt the safe area?

Reply:
Good catch!
No it would be inherited. :)

Each page (read `Scaffold`) should be wrapped into their respective `SafeArea` if its needed.
I refactored `HomePage` to have a SafeArea and a Scaffold.

Changes on [4b3a8a8](https://github.com/rishFilet/swipe_n_swap/pull/2/commits/4b3a8a80c2eb851c0ef3590e41261c3b56722a5f)